types/EdgeDataApi.ts

```diff
@@ -3,4 +3,6 @@ export type EdgeDataApi = {
   projectId: string;
   source: string; // Source node ID
   target: string; // Target node ID
+  edgeValue?: string;
```

Comment:
Is it possible to change this to label? Then we will coincide with the terms reactflow uses for text on string, which will save us from having to reassign variables.

Reply:
There is already a label value on the edges that is used in the API. It is used as categorization of the edges. 

The categories are:

```
 public const string LineEdge = "LineEdge";
        public const string ConnectedLineEdge = "ConnectedLineEdge";
        public const string OrderEdge = "OrderEdge";
        public const string TaskEdge = "TaskEdge";
        public const string SolvedTaskEdge = "SolvedTaskEdge";
```

only the two first are in use, however.

Also, the term "label" is being used a lot in both the backend and frontend for other properties. And calling this a label as well just adds more uncertainty as to what a "label" property refers to.

All in all I prefer the term "label" and i was initially using it. But it cant be called Label in the API layer unless we refactor the backend, i think.